test/unit/utils_test.js

```diff
@@ -437,6 +437,41 @@ describe('Utils', function() {
       );
     });
 
+    it("throws an error if operation value is null", function() {
+      let keypair = StellarSdk.Keypair.random();
+      const account = new StellarSdk.Account(keypair.publicKey(), "-1");
+      const transaction = new StellarSdk.TransactionBuilder(
+        account,
+        txBuilderOpts,
+      )
+        .addOperation(
+          StellarSdk.Operation.manageData({
+            name: "SDF auth",
+            value: null,
```

Comment:
Oh I see, `manageData` must see the `null` value passed and not assign `operation.value`, leaving it `undefined`.